src/plugins/intel_cpu/tests/functional/behavior/ov_plugin/properties.cpp

```diff
@@ -153,6 +153,16 @@ TEST_F(OVClassConfigTestCPU, smoke_PluginSetConfigAffinity) {
     ASSERT_EQ(affinity, value);
 }
 
+TEST_F(OVClassConfigTestCPU, smoke_PluginSetConfigAffinityCore) {
+    ov::Core ie;
+    ov::Affinity affinity = ov::Affinity::CORE;
```

Comment:
Please add test for HYBRID_AWARE and NUMA as well

Reply:
Ok, added.